Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -420,6 +426,87 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
     func numberOfSectionsInTableView(_ tableView: UITableView) -> Int {
         return SearchListSection.Count
     }
+
+    func tableView(_ tableView: UITableView, didHighlightRowAt indexPath: IndexPath) {
+        guard let section = SearchListSection.init(rawValue: indexPath.section) else {
+            return
+        }
+
+        if section == .bookmarksAndHistory,
+            let suggestion = data[indexPath.item] {
+            searchDelegate?.searchViewController(self, didHighlightText: suggestion.url, search: false)
+        }
+    }
+}
+
+extension SearchViewController {
+
+    override var canBecomeFirstResponder: Bool {
+        return true
+    }
+
+    func handleKeyCommands(sender: UIKeyCommand) {
+        let initialSection = SearchListSection.bookmarksAndHistory.rawValue
```

Comment:
We might change this value when we get we get to [Bug 1433476][1].

[1]: https://bugzilla.mozilla.org/show_bug.cgi?id=1433476